sample/gradle.properties

```diff
@@ -20,3 +20,4 @@ org.gradle.parallel=true
 org.gradle.configureondemand=true
 org.gradle.jvmargs=-Xmx3072m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
 org.gradle.caching=true
+kotlin.native.cacheKind=none
```

Comment:
Is this part of the missing feature?

Reply:
I'll document it in the PR description

Reply:
I also removed the kotlin native caching as it causes an error of missing system library when upgrading from kotlin 1.3.60 to 1.3.70. Once it work, you can remove this. As a user that wants to upgrade sample application, this prevents users from clearing their gradle native cache just for updating the sample application.